src/main/java/org/desarrolladorslp/technovation/controller/MessageController.java

```diff
@@ -47,6 +47,16 @@ public void markMessageAsLowPriority(@PathVariable UUID messageId){
         messageService.markMessageAsLowPriority(messageId);
     }
 
+    @PutMapping("/{messageId}/received")
+    public void confirmMessageReceived(@PathVariable UUID messageId){
```

Comment:
Please pass the user in this method to validate that the message actually belongs to the user making the request. Otherwise return a 403 status code.

Reply:
By the way I missed that validation in all other endpoints very similar to this one .. could you add it please?